dataland-frontend/src/components/general/ToggleChipFormInputs.vue

```diff
@@ -0,0 +1,55 @@
+<template>
```

Comment:
Question: Is the single request page accesible in the frontend? I wasn't able to see the button "request data" on a company page